functions/src/screenshot.ts

```diff
@@ -21,6 +21,9 @@ async function getBrowser(): Promise<puppeteer.Browser> {
           height: BROWSER_HEIGHT,
         },
         headless: true,
+        // Options to speed up chromium.
+        // See https://github.com/puppeteer/puppeteer/issues/1718#issuecomment-397532083
+        args: ["--proxy-server='direct://'", "--proxy-bypass-list=*"],
```

Comment:
Heh, this seems like some weird black magic.  That comment links to https://github.com/codeceptjs/CodeceptJS/issues/561#issuecomment-373666779 which says, "I dunno in which forum it was but this did the trick for me" 🤷

Reply:
yeah true 😭 I'm willing to roll this back if we'd prefer, but it does seem to help somewhat